target/xilinx/constraints/vcu118.xdc

```diff
@@ -0,0 +1,199 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Nicole Narr <narrn@student.ethz.ch>
+# Christopher Reinwardt <creinwar@student.ethz.ch>
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+# Paul Scheffler <paulsc@iis.ee.ethz.ch>
+
+#############
+# Sys Clock #
+#############
+
+# 125 MHz input clock
+set_property -dict {LOC AY24 IOSTANDARD LVDS} [get_ports sys_clk_p]
+set_property -dict {LOC AY23 IOSTANDARD LVDS} [get_ports sys_clk_n]
+create_clock -period 8.000 -name clk_125mhz [get_ports sys_clk_p]
+
+# SoC clock is generated by clock wizard and its constraints
+set_property CLOCK_DEDICATED_ROUTE BACKBONE [get_nets i_clkwiz/inst/clk_50]
+
+
+#######
+# MIG #
+#######
+
+# Dram axi clock : 333 MHz (defined by MIG constraints)
+
+# False-path incoming reset
+set_false_path -setup -hold -through [get_pins i_dram_wrapper/i_dram/c0_ddr4_aresetn]
+
+# Constrain outgoing reset
+set_false_path -hold -through [get_pins i_dram_wrapper/i_dram/c0_ddr4_ui_clk_sync_rst]
+set_max_delay -through [get_pins i_dram_wrapper/i_dram/c0_ddr4_ui_clk_sync_rst] 3.000
+
+# Limit delay across DRAM CDC (hold already false-pathed)
+# tclint-disable line-length
+set_max_delay -datapath_only -from [get_pins i_dram_wrapper/gen_cdc.i_axi_cdc_mig/i_axi_cdc_*/i_cdc_fifo_gray_*/*reg*/C] -to [get_pins i_dram_wrapper/gen_cdc.i_axi_cdc_mig/i_axi_cdc_*/i_cdc_fifo_gray_*/*i_sync/reg*/D] 3.000
+set_max_delay -datapath_only -from [get_pins i_dram_wrapper/gen_cdc.i_axi_cdc_mig/i_axi_cdc_*/i_cdc_fifo_gray_*/*reg*/C] -to [get_pins i_dram_wrapper/gen_cdc.i_axi_cdc_mig/i_axi_cdc_*/i_cdc_fifo_gray_*/i_spill_register/spill_register_flushable_i/*reg*/D] 3.000
```

Comment:
Is there a particular reason to remove $MIG_TCK (see genesys2.xdc)? Working with the variable is practical to show what may or may not change based on the board chosen.

Reply:
I'll fix it.